src/entities/cinema/api/types.ts

```diff
@@ -0,0 +1,12 @@
+export interface GetCinemaDto {
+    id: string;
+    createdAt: string;
+    updatedAt: string;
+    name: string;
+    city: {
```

Comment:
It looks like cinema is used in many places, let's create separate entity for it

Reply:
discussed